sui_core/src/authority_server.rs

```diff
@@ -168,6 +169,12 @@ impl Validator for AuthorityServer {
         Ok(tonic::Response::new(info))
     }
 
+    // // Tally requests by "route" or SerializedMessage enum name
+    // self.metrics
+    //     .requests_by_route
+    //     .with_label_values(&[message.enum_name()])
+    //     .inc();
+
```

Comment:
Did you mean to leave this comment in?

Reply:
Oh I can delete it.  It was left by accident.